docs/develop/installation.rst

```diff
@@ -16,23 +16,25 @@ It is installed from **pip** on all platforms:
 
 **Installation notes:**
 
-* Mac and Linux require you prefix the command with ``sudo``.
+* Install `dronekit` with `pip` inside a virtualenv:
     
   .. code-block:: bash
 
-      sudo pip install dronekit
+      python3 -m venv .venv
+      . .venv/bin/activate
+      pip install dronekit

```

Comment:
You've specified a version for Python but not for pip.  Probably want to be consistent.

Reply:
Once the virtualenv is activated, `pip` and `pip3` always point to the version of `pip` inside the virtualenv.